initializr-docs/src/main/asciidoc/configuration-guide.adoc

```diff
@@ -493,11 +493,13 @@ The same project can now be generated with `dependencies=ws` or
 [[dependencies-facet]]
 ==== Facets
 A "facet" is a label on a dependency which is used to drive a code modification in the
-generated project. In the standard Initializr generator, there is only one facet that is
-actually used (`web`), but custom installations might choose to use it for their own
-purposes. The `web` facet is used to drive the inclusion of `spring-boot-starter-web` if
-any other dependency with that facet is included. The value of the "facets" property of a
-dependency is a list of strings.
+generated project. In the standard Initializr generator, there are only two facets that
+are actually used (`web` and `json`), but custom installations might choose to use it for
```

Comment:
I wouldn't update the documentation of the library for a service-specific feature.